packages/functionals/botpress-qna/src/providers/microsoft_qna_maker.js

```diff
@@ -0,0 +1,171 @@
+import _ from 'lodash'
+import axios from 'axios'
+import ms from 'ms'
+import Promise from 'bluebird'
+
+const KNOWLEDGEBASE_NAME = 'botpress'
```

Comment:
Should we make it optionally configurable? Like if they're using a single account for multiple BP bots

Reply:
Makes sense. Made it configurable.